app/controllers/authors_controller.rb

```diff
@@ -0,0 +1,38 @@
+require 'net/http'
+require 'json'
+require 'uri'
+
+class AuthorsController < ActionController::Base
+  def new
+    @author = Author.new
+  end
+
+  def create
+    @author = Author.new(author_params)
+    uri = URI.parse('https://www.boredapi.com/api/activity') # simple API returning some random activity
+    if @author.valid?
+      Thread.new do
```

Comment:
There is an `ApplicationJob` for asynchronous processing in Rails. Let's use it and handle errors (connect, timeout). I also expect you to add tests for the job too. You can use Faraday library for POST request.